src/app/src/components/About.js

```diff
@@ -1,25 +1,60 @@
 import Carousel from 'nuka-carousel';
+import { Flex } from 'rebass';
 import { PagingDots } from 'nuka-carousel/lib';
 import { func, number } from 'prop-types';
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import styled from 'styled-components';
 import { Heading } from './custom-styled-components';
+import { themeGet } from 'styled-system';
 
 import { saveAboutSlideIndex, resetTimer } from '../actions';
 import { ABOUT_PROFILES } from '../util/constants';
 
 import AboutSlide from './AboutSlide';
 
 // Amounts greater than 1 indicate how much of the surrounding slides to show
-const slidesToShow = 1.3;
+const slidesToShow = 1;
```

Comment:
Curious about this change, not to show the peeking?

Reply:
The peeking is still showing without it being 1.3 🤷‍♀️

But the main issue was that two slides were getting .slide-visible as a result. When I changed the number to 1, only the slide in the middle was getting the correct class.